pallets/token-manager/src/mock.rs

```diff
@@ -376,6 +377,10 @@ impl ProcessedEventsChecker for TestRuntime {
     fn check_event(event_id: &EthEventId) -> bool {
         return PROCESSED_EVENTS.with(|l| l.borrow_mut().iter().any(|event| event == event_id))
     }
+    
+    fn add_event(event_id: &EthEventId, processed: bool) -> DispatchResult {
```

Comment:
do we need an overwrite of the default one?

Reply:
It is not possible to implement only one of the trait items without the other, this change was to resolve errors occurring as a result of adding trait functions